4-WebApp-your-API/4-1-MyOrg/README.md

```diff
@@ -1,224 +1,292 @@
+  1. Click **Register** to create the application.
+  1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+
+  1. Click **Save** to save your changes.
+
+  1. In the app's registration screen, select the **Expose an API** blade to the left to open the page where you can declare the parameters to expose this app as an API for which client applications can obtain [access tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens) for. The first thing that we need to do is to declare the unique [resource](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow) URI that the clients will be using to obtain access tokens for this API. To declare an resource URI, follow the following steps:
+      * Select `Set` next to the **Application ID URI** to generate a URI that is unique for this app.
+      * For this sample, accept the proposed Application ID URI (`api://{clientId}`) by selecting **Save**.
+  1. All APIs have to publish a minimum of one [scope](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-authorization-code) for the client's to obtain an access token successfully. To publish a scope, follow these steps:
+     * Select **Add a scope** button open the **Add a scope** screen and Enter the values as indicated below:
+          * For **Scope name**, use `ToDoList.Read`.
+          * Select **Admins and users** options for **Who can consent?**.
+          * For **Admin consent display name** type `Access TodoListService-aspnetcore-webapi`.
+          * For **Admin consent description** type `Allows the app to access TodoListService-aspnetcore-webapi as the signed-in user.`
+          * For **User consent display name** type `Access TodoListService-aspnetcore-webapi`.
+          * For **User consent description** type `Allow the application to access TodoListService-aspnetcore-webapi on your behalf.`
+          * Keep **State** as **Enabled**.
+          * Select the **Add scope** button on the bottom to save this scope.
+     * Select **Add a scope** button open the **Add a scope** screen and Enter the values as indicated below:
+          * For **Scope name**, use `ToDoList.Write`.
+          * Select **Admins and users** options for **Who can consent?**.
+          * For **Admin consent display name** type `Access TodoListService-aspnetcore-webapi`.
+          * For **Admin consent description** type `Allows the app to access TodoListService-aspnetcore-webapi as the signed-in user.`
+          * For **User consent display name** type `Access TodoListService-aspnetcore-webapi`.
+          * For **User consent description** type `Allow the application to access TodoListService-aspnetcore-webapi on your behalf.`
+          * Keep **State** as **Enabled**.
+          * Select the **Add scope** button on the bottom to save this scope.
```

Comment:
Can we try to code so that it just prints. Please follow the steps provided above to create scope(s) **ToDoList.Write** [+ more if there are more scopes in the sample.json]

Reply:
coded